shopping-cart.html

```diff
@@ -215,251 +215,251 @@ <h2 class="can-interes__title">Te podría interesar </h2>
       <div class="can-interest__background">
         <div class="can-interest__content">
           <div class="can-interest__columns">
-            <div class="can-interest__product js-can-interest-product">
-              <div class="can-interest__card">
-                <a href="#" class="can-interest__card-link"></a>
+            <ul class="can-interest__product js-can-interest-product">
+              <li class="can-interest__card">
+                <a href="#" class="can-interest__card-link" aria-label="vizualizar el producto"></a>
```

Comment:
Que producto visualiza? Tienes varios enlaces que dicen lo mismo por lo cual el usuario pensaria que se devolvio al principio al volver a leer lo mismo
```suggestion
                <a href="#" class="can-interest__card-link" aria-label="ver detalles del producto removedor de callos westinghouse"></a>
```